Google.Api.Generator.Tests/ProtoTests/Keywords/Testing.Keywords.Snippets/KeywordsClientSnippets.g.cs

```diff
@@ -30,7 +30,7 @@ public void Method1_RequestObject()
             // Initialize request argument(s)
             Request request = new Request
             {
-                EventAsResourceName = new ResourceName("[ITEM_ID]"),
+                EventAsResourceName = ResourceName.CreateItem("[ITEM_ID]"),
```

Comment:
I think we should still use the constructor for snippets of single-pattern resources. I think it's the more idiomatic form, and will still be *valid* even if the name contains multiple patterns later. I do think it's worth using the factory method for multi-pattern resources, and I don't think it's a problem for the snippets to change at that point.

Reply:
Not changed, as discussed.